### DISCUSSION TITLE
Process community rewards cycle: 11/19 - 12/16

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#372] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):

```
# NOTE ASA Stats Community rewards 11/19 - 12/16
AlgoRhythMatic 0.09 damo
BartleDooDerp  0.15 damo
DK+-x 0.03 damo
DragMZ 0.03 damo
GamingToups 0.03 damo
kerrilija 0.3 damo
Maru the Cat 0.18 damo
moloch10 0.12 damo
Reems 0.03 damo
SCN9A 0.06 damo
sealab2022 0.03 damo
UncleDooom 0.06 damo
TOTAL 1.11 damo
```
Please review and comment if there are any questions or concerns. This period uses a 3x coefficient to multiply measurements from the [Rewards Model tab](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=1130085233) from our spreadsheet.

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by ipaleka** [comment]
Damo had a testing session with Harsh for the new Algoscan app, I'm going to add him and 0.2 damo to this list.

```
# NOTE ASA Stats Community rewards 11/19 - 12/16
AlgoRhythMatic 0.09 damo
BartleDooDerp  0.15 damo
DK+-x 0.03 damo
DragMZ 0.03 damo
GamingToups 0.03 damo
kerrilija 0.3 damo
Maru the Cat 0.18 damo
moloch10 0.12 damo
Reems 0.03 damo
SCN9A 0.06 damo
sealab2022 0.03 damo
UncleDooom 0.06 damo
Damo 0.2 damo
TOTAL 1.31 damo
```
